Add `Skin of the Year 2024: Results` news post (#12945)
ppy/osu-wiki: news/2025/2025-03-12-skin-of-the-year-2024-results.md

---
layout: post
title: "Skin of the Year 2024: Results"
date: 2025-03-12 18:00:00 +0000
---

> It's time to take one final turn back at the previous year, and showcase the best community-made skins of 2024!

Author: ```suggestion
2024 never ended. In fact, it's time to showcase the best community-made skins of the year!
```
this is my rebuttal

![](/wiki/shared/news/2025-03-12-skin-of-the-year-2024-results/banner.jpg)

The voting has concluded, and after tallying up a whopping **26,856** votes submitted by **4,412** different community members, we have our final top 10 list. As you will see below, it was a close battle for every single skin in this line-up, with only a dozen or so votes separating almost every single skin.

This just goes to show how much each skin in the top 10 is completely deserving of this spotlight, and the appreciation the community has for high-quality skins, regardless of the theme.

---

[![](/wiki/shared/news/2025-03-12-skin-of-the-year-2024-results/echoing-inside.jpg)](https://osu.ppy.sh/community/forums/topics/1925867)

> [Thetasigma](https://osu.ppy.sh/users/6234482) once again earned a spotlight, this time around with their [echoing inside.](https://osu.ppy.sh/community/forums/topics/1925867) skin, which not only got the #10 spot with **520** votes, but also got the #2 spot in the [Unchained skinning contest](https://osu.ppy.sh/home/news/2024-07-17-skinning-contest-unchained-results) that was run last year.

Reviewer: ```suggestion
[thetasigma](https://osu.ppy.sh/users/6234482) once again earned a spotlight, this time around with their [echoing inside.](https://osu.ppy.sh/community/forums/topics/1925867) skin, which not only got the #10 spot with **520** votes, but also got the #2 spot in the [Unchained skinning contest](https://osu.ppy.sh/home/news/2024-07-17-skinning-contest-unchained-results) that was run last year.
```

Needless to say, if you are a fan of *Honkai: Star Rail* and it's character [Robin](https://honkai-star-rail.fandom.com/wiki/Robin), you are in for a treat.

[![](/wiki/shared/news/2025-03-12-skin-of-the-year-2024-results/wangan.jpg)](https://osu.ppy.sh/community/forums/topics/2013988)

If you are an osu! gamer that also has an interest in retro technologies, [Miorine](https://osu.ppy.sh/users/12630408) has surely already earned a spot in your skin folder.

This time around they blended up some more of that 90s nostalgia juice and made [Wangan 199X](https://osu.ppy.sh/community/forums/topics/2013988), a skin themed around JDM's and street racing, all while incorporating technologies that were used during that era, and for that the community has granted them **537** votes and the #9 spot.

[![](/wiki/shared/news/2025-03-12-skin-of-the-year-2024-results/furina.jpg)](https://osu.ppy.sh/community/forums/topics/2022040)

> [Kodama](https://osu.ppy.sh/users/8382196) kicked off their skinning career with a banging start, with [Tennensui・Furina](https://osu.ppy.sh/community/forums/topics/2022040) being the first skin they submitted!

Reviewer: ```suggestion
[kodama](https://osu.ppy.sh/users/8382196) kicked off their skinning career with a banging start, with [Tennensui・Furina](https://osu.ppy.sh/community/forums/topics/2022040) being the first skin they submitted!
```

[![](/wiki/shared/news/2025-03-12-skin-of-the-year-2024-results/windosu.jpg)](https://osu.ppy.sh/community/forums/topics/1926832)

While we've already seen what happens when retro collides with osu!, [Citrusis](https://osu.ppy.sh/users/30298378) turned that idea on it's head and made [Windosu!](https://osu.ppy.sh/community/forums/topics/1926832), a skin completely based off of Windows 11.

The skin itself remains completely loyal to the Windows 11 dark mode theme, with icons, sounds, and gameplay elements all matching closely with the operating system it's based off of. After everybody's Windows updates concluded, **566** community members decided Windosu! is worth their vote, netting Citrusis the #7 spot.

[![](/wiki/shared/news/2025-03-12-skin-of-the-year-2024-results/tetos-territory.jpg)](https://osu.ppy.sh/community/forums/topics/1974835)

VOCALOID fans are eating good today, with [Yukoii](https://osu.ppy.sh/users/34419864)'s [Kasane Teto](https://vocaloid.fandom.com/wiki/Kasane_Teto) inspired skin, [Teto's Territory](https://osu.ppy.sh/community/forums/topics/1974835).

> Despite the "anime girl" theme, Teto's Territory is actually a surprisingly simplistic skin. Simple, clean elements are present across the entire skin, dominated by pleasant shades of pink and purple, making this not only a good skin to look at, but also to play with. Even though this is Yukoii's first original completed skin, they have received huge praise that ended up earning them **571** votes, and the #6 spot.

Reviewer: word variety

```suggestion
Despite the "anime girl" theme, Teto's Territory is actually a surprisingly simplistic skin. Simple, clean elements are present across all UI elements. dominated by pleasant shades of pink and purple, making this not only a good skin to look at, but also to play with. Even though this is Yukoii's first original and complete work, they have received huge praise that ended up earning them **571** votes, and the #6 spot.
```

[![](/wiki/shared/news/2025-03-12-skin-of-the-year-2024-results/yugen-remastered.jpg)](https://osu.ppy.sh/community/forums/topics/1999325)

[Garin](https://osu.ppy.sh/users/2130664) has awoken from their long slumber, and delivered an update to the [original YUGEN skin](https://osu.ppy.sh/community/forums/topics/365036) that we didn't know we needed.

> [YUGEN ❯REMASTERED](https://osu.ppy.sh/community/forums/topics/1999325) is as the name states, a remaster of the original skin, while still maintaining the original dim minimalist style that puts all the focus on gameplay, which is why it received so much love back in the day. With new pause and fail screens, new sounds, new mod icons and much more, the community deemed the remaster worthy of **578** votes, landing it in spot #5.

Reviewer: ```suggestion
[YUGEN ❯REMASTERED](https://osu.ppy.sh/community/forums/topics/1999325) is, as the name states, a remaster of the original skin. Nevertheless, it still maintains the original dim minimalist style that puts all the focus on gameplay, which is why it received so much love back in the day. With new pause and fail screens, new sounds, new mod icons and much more, the community deemed the remaster worthy of **578** votes, landing it in spot #5.
```

[![](/wiki/shared/news/2025-03-12-skin-of-the-year-2024-results/unofficial-deskmat-skin.jpg)](https://osu.ppy.sh/community/forums/topics/2020821)

> Another familiar face joins the party! [Cyperdark](https://osu.ppy.sh/users/9893708) has already earned skinning-related accolades by winning last years [Skin of the Year](https://osu.ppy.sh/home/news/2024-02-05-skin-of-the-year-2023-results) contest, so it isn't surprising they are showing up once again.

Reviewer: ```suggestion
Another familiar face joins the party! [Cyperdark](https://osu.ppy.sh/users/9893708) has already earned skinning-related accolades by winning last year's [Skin of the Year](https://osu.ppy.sh/home/news/2024-02-05-skin-of-the-year-2023-results) contest, so it isn't surprising they are showing up once again.
```

> After last year's release of the [official osu! deskmats](https://osu.ppy.sh/home/news/2024-02-05-skin-of-the-year-2023-results) featuring three different community designs to choose from, the only logical thing to do was combine the designs into one beautiful skin, all while maintaining a clean aesthetic on the gameplay portion. This original idea was rewarded with a whopping **713** votes, and the #4 spot.

Reviewer 2: That is the wrong newspost

Author: fixed in https://github.com/ppy/osu-wiki/pull/12945/commits/fe590ff1af97bb7b87ef5d85e32b5a5903bf7b6b

Author: fixed in https://github.com/ppy/osu-wiki/pull/12945/commits/fe590ff1af97bb7b87ef5d85e32b5a5903bf7b6b

[![](/wiki/shared/news/2025-03-12-skin-of-the-year-2024-results/phosphor.jpg)](https://osu.ppy.sh/community/forums/topics/1920571)

If it seems like we are going back in time, you might be right. Not only is [Miorine](https://osu.ppy.sh/users/12630408) being mentioned again, but we are once again seeing [Phosphor](https://osu.ppy.sh/community/forums/topics/1920571), a previous contest-winning skin, and they have also earned another badge based on the same skin!

> Continuing on with the time travelling theme, Phosphor transports you back to a time when punch cards and floppies were the cutting edge technologies, pairing that with unique gameplay elements and the overall aesthetic, it's no wonder it managed to win the hearts of **787** retro-heads, and get placed at #3.

Reviewer: ```suggestion
Continuing on with the time travelling theme, Phosphor transports you back to a time when punch cards and floppies were the cutting-edge technologies, pairing that with unique gameplay elements and the overall aesthetic, it's no wonder it managed to win the hearts of **787** retro-heads, and get placed at #3.
```

![](https://assets.ppy.sh/profile-badges/SOTY2024-Phosphor@2x.png)

[![](/wiki/shared/news/2025-03-12-skin-of-the-year-2024-results/fool-moon-night2.0.jpg)](https://osu.ppy.sh/community/forums/topics/1885740)

Yet another remaster has grabbed the hearts and minds of the community, this time around being the [FOOL MOON NIGHT 2.0](https://osu.ppy.sh/community/forums/topics/1885740) skin by [Spoo](https://osu.ppy.sh/users/11805037)!

> Spoo took the [original skin](https://osu.ppy.sh/community/forums/topics/1775136)'s idea (inspired by the iconic [FOOL MOON NIGHT](https://osu.ppy.sh/beatmapsets/524026#osu/1141084) beatmap background) and cranked it up to 11, re-drawing each element with much more detail, even incorporating memes, if you know where to look. As a result of their three months of hard work, the community overwhelmingly supported Spoo, bringing in a total of **882** votes and securing the #2 spot!

Reviewer: ```suggestion
Spoo took the [original skin](https://osu.ppy.sh/community/forums/topics/1775136)'s idea (inspired by the iconic [FOOL MOON NIGHT](https://osu.ppy.sh/beatmapsets/524026#osu/1141084) beatmap background) and cranked it up to 11, redrawing each element with much more detail, even incorporating memes, if you know where to look. As a result of their three months of hard work, the community overwhelmingly supported Spoo, bringing in a total of **882** votes and securing the #2 spot!
```

![](https://assets.ppy.sh/profile-badges/SOTY2024-FOOL-MOON-NIGHT@2x.png)

[![](/wiki/shared/news/2025-03-12-skin-of-the-year-2024-results/freedom-dive-remastered.jpg)](https://osu.ppy.sh/community/forums/topics/2019099)

You could say that 2024 was [Spoo](https://osu.ppy.sh/users/11805037)'s year of the remaster, and it looks like it has paid out it's dividends, with them taking both the #1 and #2 spots!

> With yet another throwback, the [original skin](https://osu.ppy.sh/community/forums/topics/1293207) was based on the famous [background](https://www.pixiv.net/en/artworks/41256222) used across various FREEDOM DiVE beatmapsets, and it was created all the way back in 2021 for the [first ever skinship skinning contest](https://osu.ppy.sh/home/news/2021-03-08-skinning-contest-announcement)! After it received the Spoo™ treatment with a complete re-draw of all elements, it was received with huge praise, so much so that the community deemed it worthy of the #1 spot, with **1078** votes!

Reviewer: ```suggestion
With yet another throwback, the [original skin](https://osu.ppy.sh/community/forums/topics/1293207) was based on the famous [background](https://www.pixiv.net/en/artworks/41256222) used across various FREEDOM DiVE beatmapsets, and it was created all the way back in 2021 for the [first-ever skinship skinning contest](https://osu.ppy.sh/home/news/2021-03-08-skinning-contest-announcement)! After it received the Spoo™ treatment with a complete redraw of all elements, it was received with huge praise, so much so that the community deemed it worthy of the #1 spot, with **1078** votes!
```

> With yet another throwback, the [original skin](https://osu.ppy.sh/community/forums/topics/1293207) was based on the famous [background](https://images6.alphacoders.com/974/974020.png) used across various FREEDOM DiVE beatmapsets, and it was created all the way back in 2021 for the [first ever skinship skinning contest](https://osu.ppy.sh/home/news/2021-03-08-skinning-contest-announcement)! After it received the Spoo™ treatment with a complete re-draw of all elements, it was received with huge praise, so much so that the community deemed it worthy of the #1 spot, with **1078** votes!

Reviewer 3: ```suggestion
With yet another throwback, the [original skin](https://osu.ppy.sh/community/forums/topics/1293207) was based on the famous [background](https://www.pixiv.net/en/artworks/41256222) used across various FREEDOM DiVE beatmapsets, and it was created all the way back in 2021 for the [first ever skinship skinning contest](https://osu.ppy.sh/home/news/2021-03-08-skinning-contest-announcement)! After it received the Spoo™ treatment with a complete re-draw of all elements, it was received with huge praise, so much so that the community deemed it worthy of the #1 spot, with **1078** votes!
```
Might be better to at least link it to the original artists' page as it is the background used for the Cytus mobile game's inclusion of FREEDOM DiVE.

Author: who's the original artist? i can just find various wiki pages from other rhythm games

Reviewer 3: original artist was linked to be [SnowEgg](https://www.pixiv.net/en/users/1755315) who did work on various Rayark background covers like [AXION](https://www.pixiv.net/en/artworks/37924740), [Rainbow Night Sky Highway](https://www.pixiv.net/en/artworks/37992095) and various [DeeMo](https://www.pixiv.net/en/artworks/56764190) / [VOEZ](https://www.pixiv.net/en/artworks/57244675) tracks.

Author: i'll just link to https://www.pixiv.net/en/artworks/41256222

![](https://assets.ppy.sh/profile-badges/SOTY2024-FREEDOM-DiVE-REiMAGINED@2x.png)

---

The top 3 entries will all receive a unique profile badge based on their entry along with 4, 3 and 2 months of osu!supporter respectively, and entries #4–10 will all receive a month of osu!supporter each.

As always, you can view all submitted skins [here](https://osu.ppy.sh/community/contests/239), along with their vote counts, so you can check where your favourite skins landed on the leaderboard.

> Thank you to the entire skinning community for taking their time out of their days to create unique, amazing skins for the entire community to enjoy! They have expressed so much creativity, not just this year, but in general, always striving to make their ideas become reality, and sharing their passion for art and design.

Reviewer: sounds a bit nicer

```suggestion
Thank you to the entire skinning community for taking the time out of your days to create unique, amazing skins for the entire community to enjoy! You have all expressed so much creativity, not just this year, but in general, always striving to make your ideas become reality and share your passion for art and design.
```
